Combine Legal and Conformance section (#119)
ietf-wg-aipref/drafts: draft-ietf-aipref-vocab.md

The process of obtaining preferences has very limited scope
for variation between implementations.

## Applicability and Effect {#applicability}

This specification provides a set of definitions for different 
categories of use, plus a system for associating simple 
preferences to each (allow, disallow, or no preference; see {{model}}).

This specification does not provide any enforcement mechanism 
for those preferences, and conformance to it does not encompass 
whether preferences are actually respected during data processing.

Preferences do not themselves create rights or prohibitions, 
either in the positive or the negative. Other mechanisms—technical, 
legal, contractual, or otherwise—might enforce stated preferences 
and thereby determine the consequences of following or not following 
a stated preference.

> A data processor MAY choose to respect preferences that it has 
> discovered, according to

Reviewer: ```suggestion
An entity that receives usage preferences MAY choose to respect
those preferences it has discovered, according to
```

an understanding of how the asset is used,
how that usage corresponds to the usage categories
where preferences have been expressed,
and the applicable legal context.

Usage preferences can be overridden through express agreements 
between relevant parties, by explicit provisions of law, or 
through the exercise of discretion in situations where other 
priorities justify doing so. Priorities that could justify 
ignoring preferences include—but are not limited to—free 
expression, safety, education, scholarship, research, 
preservation, interoperability, and accessibility.

The following lists examples of cases

> where other priorities could override specific preferences:

Reviewer: I think that I prefer Mark's take on this list of things, both the framing and the "might be allowed by a legal system to ignore a preference"

Author: you mean replacing lines 198 and 199 with: 

> For example, a legal system might require adherence to stated preferences,
but allow them to be ignored for specified purposes, such as:

Reviewer 2: I prefer @paul2keller s proposed language insofar as it doesn't refer to a legal system that allows ignoring a certain preference. By default, anything that is not expressly prohibited by a legal system is allowed. Therefore, a legal system rarely is explicit in what it allows. Consequently, "a legal system might ... allow them to be ignored" could create misunderstandings where people might read this as requiring an explicit legal provision that allows somebody to ignore a stated preference. In reality, unless a legal system explicitly requires somebody to _respect_ a stated preference, ignoring it is allowed.

The process of obtaining preferences has very limited scope
for variation between implementations.

> ## Applicability and Effect 

Reviewer: ```suggestion
## Applicability and Effect {#applicability}
```

This specification provides a set of definitions for different 
categories of use, plus a system for associating simple 
preferences to each (allow, disallow, or no preference; see {{model}}).

This specification does not provide any enforcement mechanism 
for those preferences, and conformance to it does not encompass 
whether preferences are actually respected during data processing.

Preferences do not themselves create rights or prohibitions, 
either in the positive or the negative. Other mechanisms—technical, 
legal, contractual, or otherwise—might enforce stated preferences 
and thereby determine the consequences of following or not following 
a stated preference.

A data processor MAY choose to respect preferences that it has 

> discovered, according to:
> 
> *	an understanding of the nature of the processing being performed 
> and how it corresponds to the usage categories where preferences 
> have been expressed, and
> * the applicable legal context.

Reviewer: A list of two might just be better as a sentence:

```suggestion
discovered, according to
an understanding of how the asset is used,
how that usage corresponds to the usage categories
where preferences have been expressed,
and the applicable legal context.
```

  in order to develop or use tools that detect harmful content
  according to established terms of use.

Because enforcement is not provided by this specification,
the consequences of ignoring preferences may vary

> depending upon how a given legal jurisdiction recognizes preferences;
> see {{applicability}}.

Reviewer: ```suggestion
depending upon how a given legal jurisdiction recognizes preferences.
```

  in order to develop or use tools that detect harmful content
  according to established terms of use.

> Because enforcement is not provided by this specification, the consequences of ignoring preferences may vary depending upon how a given legal jurisdiction recognizes preferences, including how it maps categories of use to specific applications.

Reviewer: ```suggestion
Because enforcement is not provided by this specification,
the consequences of ignoring preferences may vary
depending upon how a given legal jurisdiction recognizes preferences;
see {{applicability}}.
```
